cmd/ttn-lw-cli/commands/end_devices_split.go

```diff
@@ -339,3 +340,46 @@ func deleteEndDevice(ctx context.Context, devID *ttnpb.EndDeviceIdentifiers) err
 
 	return nil
 }
+
+func updateDeviceLocation(device *ttnpb.EndDevice, flags *pflag.FlagSet) bool {
+	latitude, _ := flags.GetFloat64("location.latitude")
+	longitude, _ := flags.GetFloat64("location.longitude")
+	altitude, _ := flags.GetInt32("location.altitude")
+	accuracy, _ := flags.GetInt32("location.accuracy")
+
+	if latitude == 0 && longitude == 0 && altitude == 0 && accuracy == 0 {
+		return false
+	}
+	// need both latitude and longitude
+	if latitude == 0 {
+		logger.Warn("--location.latitude is required")
+		return false
+	}
+	if longitude == 0 {
+		logger.Warn("--location.longitude is required")
+		return false
+	}
+
+	if device.Locations == nil {
+		device.Locations = make(map[string]*ttnpb.Location)
+	}
+	loc, ok := device.Locations["user"]
+	if !ok {
+		loc = &ttnpb.Location{}
+	}
+	loc.Source = ttnpb.SOURCE_REGISTRY
+	if altitude != 0 {
+		loc.Altitude = altitude
+	}
+	if longitude != 0 {
+		loc.Longitude = longitude
+	}
+	if latitude != 0 {
+		loc.Latitude = latitude
+	}
+	if accuracy != 0 {
+		loc.Accuracy = accuracy
+	}
```

Comment:
This would mean you can't update the altitude to 0, or remove the accuracy. Why's that?